### PR TITLE
Fixed #34628 -- Improved icons accessibility of related widget wrapper in admin.

### PR DESCRIPTION
**Pull request by nmenezes0**
Fixes https://code.djangoproject.com/ticket/34628.

Makes icons larger, greys them out when disabled.

### CODE REVIEW

File: django/contrib/admin/static/admin/css/widgets.css [hunk 1]
```diff
@@ -579,11 +579,13 @@ ul.timelist, .timelist li {
...
+    filter: grayscale(1);
 }
 
 .related-widget-wrapper-link:link {
-    opacity: .8;
+    opacity: 1;
+    filter: grayscale(0);
 }
 
 .related-widget-wrapper-link:link:focus,
```
Comment (thibaudcolas) [inline django/contrib/admin/static/admin/css/widgets.css:591]: Those hover / focus styles should be removed if we always have full opacity for links.

File: django/contrib/admin/templates/admin/widgets/related_widget_wrapper.html [hunk 2]
```diff
@@ -32,7 +32,7 @@
           <a class="related-widget-wrapper-link view-related" id="view_id_{{ name }}"
```
Comment (thibaudcolas) [inline django/contrib/admin/templates/admin/widgets/related_widget_wrapper.html:32]: ^ Above – we also need to add the width and height for the "Delete" action link.

**Review by thibaudcolas — CHANGES_REQUESTED** [review]
Thank you for giving this a go @nmenezes0! The visual changes look good to me. Having bigger, less transparent icons will be great for everyone. In the future, please make sure to add a more descriptive title for your pull request and ideally include a screenshot (or before/after) in the description so it’s easier to understand the changes you made.

There is a further refactoring to implement, and an extra icon to update.

**Comment by nmenezes0** [comment]
> Thank you for giving this a go @nmenezes0! The visual changes look good to me. Having bigger, less transparent icons will be great for everyone. In the future, please make sure to add a more descriptive title for your pull request and ideally include a screenshot (or before/after) in the description so it’s easier to understand the changes you made.
> 
> There is a further refactoring to implement, and an extra icon to update.

Thanks - will have a go at those changes, and will add more detail next time I create a PR.

File: django/contrib/admin/templates/admin/widgets/related_widget_wrapper.html (hunk 2 above)
Comment (nmenezes0) [reply django/contrib/admin/templates/admin/widgets/related_widget_wrapper.html:32]: I have added the width and height, though I got a bit stuck trying to test (trying to find out when the delete should appear).

By doing something hacky, this is the before:
image

And this is the after:
image

File: django/contrib/admin/static/admin/css/widgets.css (hunk 1 above)
Comment (nmenezes0) [reply django/contrib/admin/static/admin/css/widgets.css]: Now have been removed.

**Review by felixxm — APPROVED** [review]
@nmenezes0 Thanks :+1: I squashed commits and added a small test for `width`/`height`.

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Comment by nmenezes0** [comment]
> 

Thanks @felixxm and thanks @thibaudcolas for all your help with this!

**Pull request merged**